src/ocamlorg_frontend/pages/outreachy.eml

```diff
@@ -10,24 +10,29 @@ Layout.render
         <p>This is a record of all past OCaml Community <a href="https://www.outreachy.org/" target="_blank">Outreachy</a> Internship Projects.</p>
     </div>
 </div>
+
 <div class="py-12 bg-default dark:bg-dark-default">
     <div class="container-fluid">
         <% metadata |> List.iter (fun (rounds : Data.Outreachy.t) -> %>
             <h2 class="border-b border-primary-700 p-6 my-2 font-bold"><%s rounds.name %></h2>
             <div class="grid gap-10 prose max-w-4xl">
                 <% rounds.projects |> List.iter (fun (project : Data.Outreachy.project) -> %>
-                    <div x-data="{'is_shown': false}" class="p-5 flex flex-col gap-2">              
+                    <div x-data="{'is_shown': false}" class="p-5 flex flex-col">              
+                        
                         <h3><%s project.title %></h3>
+
                         <div>Mentee: <i><%s project.mentee %></i></div>
+                        
                         <div>Mentor(s): <i><%s Fmt.(str "%a" (list ~sep:comma string) project.mentors) %></i></div>
+                        
                         <% if String.length project.description > 240 then ( %>
                             <div :class="is_shown ? '' : 'relative max-h-20 overflow-hidden'">
                                 <div class="absolute bottom-0 h-10 left-0 right-0 bg-gradient-to-t from-white"></div> 
-                                <p> <%s project.description %> </p>
+                                <p class="overflow-fix"> <%s project.description %> </p>
```

Comment:
`overflow-fix` doesn't seem defined anywhere and I don't think this file needs to be changed when we remove the offending links from the outreachy descriptions.

Reply:
Oh, I added it during debugging and forgot to remove it when I found the bug. Do you need me to take it out or is that already done?

Reply:
when you look at the "Files changed" tab, you'll see that two files are being edited by this PR. Best is to check out the old version of `outreachy.eml`, so the PR doesn't make any changes to this file.